### PR TITLE
Python Discord Bot: Basic commands and Music

### PR DESCRIPTION
**Pull request by nishantc7**
Initial Commit. Created Bot.py and a COG music.py. Updated README.md for requirements and commands.

### CODE REVIEW

**Comment by Zanark** [comment]
@nishantc7 make sure to add a description next time(better, edit this one now) and give a proper title to your PRs

File: README.md [hunk 1]
```diff
@@ -1,9 +1,38 @@
 # codex-discord-bot
 CODEX bot for the CODEX Discord channel
+[![Gitter](https://badges.gitter.im/codex-discord-bot/community.svg)](https://gitter.im/codex-discord-bot/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 # MAINTAINERS
 - [Debashish Mishra](https://github.com/Zanark)
 
-# CONTRIBUTORS
+# Requirements
```
Comment (Zanark) [inline README.md]: Don't move the Contributors list, add the new stuff after the Contributors list.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
+
+## To get your Youtube API Token follow the instructions in the links provided 
+https://developers.google.com/youtube/v3/getting-started
+To create your project and credentials 
+https://console.cloud.google.com
+
+### Commands
+
+There are many commands that can be used with the bot.
+- !ping : Returns pong
```
Comment (Zanark) [inline README.md]: pong?

**Review by Zanark — CHANGES_REQUESTED** [review]
@nishantc7 I have reviewed your PR. Please make changes as requested.

File: README.md (hunk 2 above)
Comment (nishantc7) [reply README.md]: Updated.

**Pull request merged**